README.md

```diff
@@ -0,0 +1,10 @@
+Bykr
+====
+
+Installation
```

Comment:
Your documentation could use a bit of work. I might save this for a separate PR.